### PR TITLE
KTLN-131 Kotlin Spring Security DSL Initial Commit

### PR DESCRIPTION
**Pull request by musibs**
Pull request to merge the changes for the article KTLN-131 | Spring Security with Kotlin DSL

### CODE REVIEW

File: kotlin-spring-security-dsl/.mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/*
```
Comment (rpvilao) [inline kotlin-spring-security-dsl/.mvn/wrapper/MavenWrapperDownloader.java:1]: this seems to be here by mistake

File: kotlin-spring-security-dsl/.mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```
Comment (rpvilao) [inline kotlin-spring-security-dsl/.mvn/wrapper/maven-wrapper.properties:1]: this one too

File: kotlin-spring-security-dsl/pom.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <!--exclusions>
```
Comment (rpvilao) [inline kotlin-spring-security-dsl/pom.xml:54]: remove if not needed

File: kotlin-spring-security-dsl/.mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (musibs) [reply kotlin-spring-security-dsl/.mvn/wrapper/MavenWrapperDownloader.java]: I've removed it.

File: kotlin-spring-security-dsl/.mvn/wrapper/maven-wrapper.properties (hunk 2 above)
Comment (musibs) [reply kotlin-spring-security-dsl/.mvn/wrapper/maven-wrapper.properties]: I've removed it.

File: kotlin-spring-security-dsl/pom.xml (hunk 3 above)
Comment (musibs) [reply kotlin-spring-security-dsl/pom.xml]: I've removed it.

**Pull request merged**